jenkins/python-setup.sh

```diff
@@ -1,6 +1,6 @@
 _prefix="${1}"
 
-pip3 install --no-cache-dir --prefix="${_prefix}" -r ${progdir}/lint-requirements.txt -r ${progdir}/agent/requirements.txt -r ${progdir}/server/requirements.txt -r ${progdir}/agent/test-requirements.txt -r ${progdir}/server/test-requirements.txt
+pip3 install --no-cache-dir --progress-bar off --no-color --prefix="${_prefix}" -r ${progdir}/lint-requirements.txt -r ${progdir}/agent/requirements.txt -r ${progdir}/server/requirements.txt -r ${progdir}/agent/test-requirements.txt -r ${progdir}/server/test-requirements.txt
```

Comment:
This is a desparate attempt to try to remove the escape sequences when running in the container, but for the life of me I can't figure out why the still appear.